src/entities/person/possible-duplicates.ts

```diff
@@ -0,0 +1,82 @@
+import { Universe } from 'src/universe'
+import { EntityRawPayload, UniverseEntityOptions } from '../_base'
+import { PersonRawPayload, PersonPhonenumberRawPayload } from './person'
+import { ChannelUserRawPayload } from './channel-user'
+import { BaseError } from '../../errors'
+
+export interface PossibleDuplicatesRawPayload extends EntityRawPayload{
+  readonly person_id: string
+  readonly strategies: PossibleDuplicatesStrategies
+}
+
+interface PossibleDuplicatesStrategies {
+  global_phonenumber?: StrategiesResult[]
+  name_exact_matches?: StrategiesResult[]
+}
+
+interface StrategiesResult {
+  document: StrategiesDocument
+  kind: string
+}
+
+interface StrategiesDocument {
+  person: Pick<PersonRawPayload, 'id'| 'kind' | 'name'>
+  channel_user?: Pick<ChannelUserRawPayload, 'id' | 'name' | 'phone'>
+  phone_number?: Pick<PersonPhonenumberRawPayload, 'id' | 'value'>
+}
+
+interface PossibleDuplicationOptions extends UniverseEntityOptions {
+  rawPayload: PossibleDuplicatesRawPayload
+}
+
+export class PossibleDuplication {
+  protected universe: Universe
+  protected apiCarrier: Universe
+  protected http: Universe['http']
+  protected options: PossibleDuplicationOptions
+  public initialized: boolean
+  public endpoint: string
+
+  public id?: string
```

Comment:
Is this actually really optional?

Reply:
Right now it is not part of the mock data we're returning on the endpoint indeed :), this `id` will be one of the entries of `people.possible_duplicated_contacts_lists_items`. For now I don't return the raw entry/item just as-is from the table but the return will be a composite/results of some queries over this and other tables as well so I don't see a need right now to send the id's in the return but this might change in the future when we may have some use case where you would need to load a specific item from `people.possible_duplicated_contacts_lists_items` for a person without rechecking and therefore re-insert the entries for that person :)

TLDR; this is for now indeed optional and might change in the future then we can be more strict and remove the optional